universal/algo.py

```diff
@@ -31,15 +31,17 @@ class Algo(object):
     #    raw:    pt
     PRICE_TYPE = 'ratio'
 
-    def __init__(self, min_history=None, frequency=1):
+    def __init__(self, min_history=None, frequency=1, **kwargs):
         """ Subclass to define algo specific parameters here.
         :param min_history: If not None, use initial weights for first min_window days. Use
             this if the algo needs some history for proper parameter estimation.
         :param frequency: algorithm should trade every `frequency` periods
         """
         self.min_history = min_history or 0
-        self.frequency = frequency
-
+        self.frequency   = frequency
+        self.trx_fee_pct = kwargs.get('trx_fee_pct', 0)
+        self.n           = kwargs.get('n', 10)
```

Comment:
Could you rename it so that it's more clear it's related to transaction costs? e.g. `trx_fee_n`. Variable `n` is too general and could be confusing (and algos might use it for themselves)

Reply:
Good idea, done !